### ISSUE TITLE
ta_id: does teep-device have to match all members to 

### ISSUE THREAD

**Issue by mcd500** [ietf-teep/teep-protocol#4]
We are in the middle of prototyping the teep over http.

The current draft specifies the `ta_id` with three members.
 
https://github.com/ietf-teep/teep-protocol/blob/master/draft-ietf-teep-protocol.xml#L327

At our initial protyping we are matching all three strings when tam received the requested ta from the teep-broker.
```
Vendor_ID=ietf-teep-wg
Class_ID=uuid or ta-name
Device_ID=teep-device
```

We are not sure matching all three member all the time is the expected behaviour, since ta-name or uuid matters most of the time.

Any recommendation is helpful for us.

Or Vender_ID and Device_ID are expected as only information only?

**Comment by hannestschofenig** [comment]
In OP-TEE UUIDs are used as filenames for TAs, such as 8aaaf200-2450-11e4-abe2-0002a5d5c51b

**Comment by mcd500** [comment]
Based on the discussion at yesterday interim meeting, probably the unique identifier is good with using a hash value from both the filename (uuid.ta in optee, and somename.dll in sgx) and the version of the ta.

**Comment by dthaler** [comment]
Fixed in draft-13 (but issue #70 tracks discussion still in progress)
